AwsCryptographyPrimitives/runtimes/go/ImplementationFromDafny-go/awscryptographyprimitivessmithygenerated/api_client.go

```diff
@@ -0,0 +1,557 @@
+// Code generated by smithy-go-codegen DO NOT EDIT.
+
+package awscryptographyprimitivessmithygenerated
+
+import (
+	"context"
+
+	"github.com/aws/aws-cryptographic-material-providers-library/primitives/AtomicPrimitives"
+	"github.com/aws/aws-cryptographic-material-providers-library/primitives/AwsCryptographyPrimitivesTypes"
+	"github.com/aws/aws-cryptographic-material-providers-library/primitives/awscryptographyprimitivessmithygeneratedtypes"
+	"github.com/dafny-lang/DafnyRuntimeGo/v4/dafny"
+)
+
+type Client struct {
+	DafnyClient AwsCryptographyPrimitivesTypes.IAwsCryptographicPrimitivesClient
+}
+
+func NewClient(clientConfig awscryptographyprimitivessmithygeneratedtypes.CryptoConfig) (*Client, error) {
+	var dafnyConfig = CryptoConfig_ToDafny(clientConfig)
+	var dafny_response = AtomicPrimitives.Companion_Default___.AtomicPrimitives(dafnyConfig)
+	if dafny_response.Is_Failure() {
+		panic("Client construction failed. This should never happen")
+	}
+	var dafnyClient = dafny_response.Extract().(AwsCryptographyPrimitivesTypes.IAwsCryptographicPrimitivesClient)
+	client := &Client{dafnyClient}
+	return client, nil
+}
+
+func (client *Client) GenerateRandomBytes(ctx context.Context, params awscryptographyprimitivessmithygeneratedtypes.GenerateRandomBytesInput) ([]byte, error) {
+	err := params.Validate()
+	if err != nil {
+		opaqueErr := awscryptographyprimitivessmithygeneratedtypes.OpaqueError{
```

Comment:
We could, but it's a risk with generated code. It might collide if there's any symbol named `primitiveTypes`, and since customers aren't exposed to this, I'd keep it this way.